### ISSUE TITLE
Have a focusing gaussian beam coming out of the boundary at an angle

### ISSUE THREAD

**Issue by lasertime** [Warwick-Plasma/epoch#464]
Does anyone know how to have a focusing Gaussian beam that comes out at an angle? Additionally, be able to move the laser so it doesn't come out a 0 on the boundary? I have been trying my best using the other issues that talked about it and the EPOCH manual but haven't had any luck doing it.

**Comment by TomGoffrey** [comment]
Does #451 help at all?

**Comment by lasertime** [comment]
also, how was las_focus_time calculated and what is it for? What does the - 2 * pi * c / las_lambda * las_focus_time do?

**Comment by skuschel** [comment]
the simulation always starts at `t=0`. The laser pulse will be injected such, that at `t=las_focus_time` the maximum of the laser pulses reaches the focus. Depending on how far your boundary is from the focus position and you long your laserpulse is, you have to set the `las_focus_time` accordingly.

The value `las_focus_time` is in the phase term to adjust the injection for the correct CE-phase.

**Comment by lasertime** [comment]
so then las_focus_time would be the distance laser travels from boundary to origin (assuming i want focus to be at origin) divided by the speed of light plus the time until FWHM?

**Comment by skuschel** [comment]
yes, you should set it to at least that value. Its probably a good idea to add 3*FWHM to make sure, that the laser pulse is not cut on the rising edge.
